.github/workflows/test.yml

```diff
@@ -67,6 +71,7 @@ jobs:
             plt-cache-
 
       - name: Check function specs with Dialyzer
+        if: matrix.elixir == '1.16' && matrix.otp == '26'
```

Comment:
This wasn't causing a problem but also figured this could be skipped since the new dialyzer is probably better

Reply:
yeah nobody wants to wait for dialyzer to run twice